modules/gitops-release-notes-1-4-12.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-4-12_{context}"]
+= Release notes for {gitops-title} 1.4.12
+
+{gitops-title} 1.4.12 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-4-12_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
```

Comment:
Personally, I am happy with the passive voice here.

But if you want to avoid it, as per the style guide, you could say:

"The current release resolves the following issues:"

Reply:
@gabriel-rh keeping the passive voice, since all the previous release notes have the same.